plenum/common/messages/internal_messages.py

```diff
@@ -33,3 +33,17 @@
 
 RequestPropagates = NamedTuple('RequestPropagates',
                                [('bad_requests', List)])
+
+NodeModeMsg = NamedTuple('NodeModeMsg',
+                         [('mode', int)])
+
+PrimariesBatchNeeded = NamedTuple('PrimariesBatchNeeded',
+                                  [('pbn', bool)])
+
+CurrentPrimaries = NamedTuple('CurrentPrimaries',
+                              [('primaries', list)])
+
+TryOrderMsg = NamedTuple('TryOrderMsg',
```

Comment:
Why do we need this message? I believe `OrderingService` should be responsible for creation of `Ordered` messages and sending them to Internal Bus